lib/plugin.js

```diff
@@ -38,6 +38,7 @@ var createPreprocesor = function(logger, config, basePath) {
 
     var output =
       'window.__cjs_modules_root__ = "' + modulesRootPath + '";' +
+      (config.istanbul.ignore ? ' ' + config.istanbul.comment + ' ' : '') +
```

Comment:
If we make it required, depending on `ignore` we should either have a default or throw an error when it doesn't exist so users know what the problem is.